corehq/apps/app_manager/models.py

```diff
@@ -4591,6 +4597,10 @@ class Application(ApplicationBase, ApplicationMediaMixin, ApplicationIntegration
 
     family_id = StringProperty()  # ID of earliest parent app across copies and linked apps
 
+    def __repr__(self):
+        return (f"{self.doc_type}(id='{self._id}', domain='{self.domain}', "
+                f"name='{self.name}', is_build={bool(self.copy_of)})")
```

Comment:
```python
is_build = bool(self.copy_of)
```
^ this is a little misleading, as `is_build` isn't a real property.  Should I just include `copy_of`?  I guess if it's an actual build then that would be useful information to have.

Reply:
That's tricky. I think it would be fairly easy to find where `is_build` is coming from by just searching for it. But I wonder having the version would be helpful too or if that is cluttering already.

Reply:
just changed it to `copy_of` and also added unique IDs